Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -3058,6 +3061,45 @@ bool AccessibilityNodeObject::isDescriptionList() const
     return elementName() == ElementName::HTML_dl;
 }
 
+static bool childrenContainOnlyStaticText(const AccessibilityObject::AccessibilityChildrenVector& children)
+{
+    if (children.isEmpty())
+        return false;
+    for (const auto& child : children) {
+        if (child->role() == AccessibilityRole::StaticText)
+            continue;
+        if (child->isGroup()) {
+            if (!childrenContainOnlyStaticText(child->unignoredChildren()))
+                return false;
+        } else
+            return false;
+    }
+    return true;
+}
+
+bool AccessibilityNodeObject::labelContainsOnlyStaticText() const
```

Comment:
Something like `isLabelContaininingOnlyStaticText` might be more accurate? It also seems like we'll be duplicating the `isLabel()` check in the only place that calls this new method? Probably not the biggest deal though